Week3/homework/step2-3.js

```diff
@@ -5,8 +5,13 @@ function repeatStringNumTimesWithFor(str, num) {
   // eslint-disable-next-line
   let result = '';
 
-  // Replace this comment and the next line with your code
-  console.log(str, num, result);
+  if (num <= 0) {
+    result = '';
+  } else {
+    for (let i = 0; i < num; i++) {
+      result += str;
+    }
+  }
```

Comment:
Guess what: your code still works fine when I comment out lines 8-10 and line 14. Can you explain why?

Reply:
Because in this case `num` variable is always positive and if you comment ou that lines code will still continue to work :) I delete these parts, thanks.